metadata.json

```diff
@@ -17,5 +17,14 @@
       "name": "puppetlabs/apt",
       "version_requirement": ">= 1.4.0"
     }
+  ],
+  "operatingsystem_support": [
+    {
+      "operatingsystem": "Ubuntu",
+      "operatingsystemrelease": [
+        "12.04",
+        "14.04"
```

Comment:
this would require way more testing. I would like to keep that for a 2.0.0 release